katas/content/multi_qubit_gates/index.md

```diff
@@ -304,7 +304,7 @@ $$=|00\rangle\langle00| + |01\rangle\langle01| + |10\rangle\langle11| + |11\rang
     "title": "SWAP Gate"
 })
 
-The $SWAP$ gate acts on two qubits, and, as the name implies, swaps their quantum states.
+The $\text{SWAP}$ gate acts on two qubits, and, as the name implies, swaps their quantum states.
```

Comment:
Let's keep this as $SWAP$ for now for consistency, since it's by far not the only occurrence of a multi-letter gate in the katas. Once we agree on the style we should use for gates, we can do the update more broadly

Reply:
Done in incoming changes.